src/filters/day/day.constants.ts

```diff
@@ -1,5 +1,5 @@
 const day = '((3[0-1])|([1-2][0-9])|(0?[1-9]))';
-const ordinalIndicators = '(st|nd|rd|th)';
+const ordinalIndicators = '(st|nd|rd|th)?';
```

Comment:
Don't add`?` here, add it when you need to use ordinal indicators, see https://github.com/h-dong/Serina/blob/master/src/filters/dates/dates.constants.ts#L10

Reply:
This is just constant, we don't want to make the assumption that it is always optional

Reply:
I can agree on not putting it here. But I think the example you have given is confusing. It makes it seem that the day itself is optional (without seeing the expression in full).

Reply:
ANY makes it seem that it should accept any date, with or without ordinal. I would then refactor ANY to have optional ordinal and have another one for ordinal only. What do you think?

Reply:
Yeah let's go with what you said last